model-optimizer/extensions/middle/FusedBatchNormTraining.py

```diff
@@ -44,15 +46,14 @@ class FusedBatchNormTraining(MiddleReplacementPattern):
     def pattern(self):
         return dict(
             nodes=[
-                ('op', dict(kind='op', op=lambda op: op in ['FusedBatchNorm', 'FusedBatchNormV2', 'FusedBatchNormV3'],
-                 is_training=True))],
+                ('op', dict(kind='op', op=lambda op: op in ['batchNormTraining']))],
             edges=[]
         )
 
     def replace_pattern(self, graph: Graph, match: dict):
```

Comment:
Refactor this transformation to the following approach (and remove `pattern` function):

```python
    def find_and_replace_pattern(self, graph: Graph):
        for batch_norm_training_node in graph.get_op_nodes(op='batchNormTraining'):
```